### PR TITLE
docs: Update guide additional steps for 16.1.x to 17.0.x

### LINKED ISSUE #3972: Add migration guide for breaking change in 17.0
- for combo
- also for  
> `animations` are now imported from `igniteui-angular/animations`/`@infragistics/igniteui-angular/animations`

### PR DESCRIPTION
**Pull request by wnvko**
Add notes in update guide about breaking changes in combo and simple combo.

Closes #3972 

### Checklist:
 
 - [ ] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [ ] at the References section at the end of the topic add links to topics, samples, etc
 - [ ] reference API documentation instead of adding a section with API

<br />

 - [x] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [x] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [x] add inline `code blocks` for the names of classes / tags / properties
 - [x] add language descriptor for the ```code blocks```
 - [ ] check broken links (use browser add-on)
 - [ ] check if sample is working and fully visible in the topic
 - [ ] check if sample is working and fully visible in the StackBlitz
 - [ ] check if code blocks match the code in StackBlitz demo

<br />

 - [ ] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [x] add `pending-localization` label when the review of the PR is done
 - [x] add a member from the localization team to translate it

### CODE REVIEW

File: en/components/general/update-guide.md [hunk 1]
```diff
@@ -51,6 +51,44 @@ Unfortunately not all changes can be automatically updated. Changes below are sp
...
 
+## From 16.1.x to 17.0.x
+
+- **Breaking change**
+- `selectionChanging` arguments type is changed. Now the `oldSelection`, `newSelection`, `added` and `removed` collections, part of the `IComboSelectionChangingEventArgs` interface, no longer consist of the keys of the selected items (when the combo has set a `valueKey`), but now in any case the items are emitted. When the combo is working with remote data and a `valueKey` is set - for the selected items that are not currently part of combo view, a partial item data object will be emitted.
+
+If your code in `selectionChanging` event handler was depending on reading `valueKeys` from the event argument, update it as follows:
+
+```typescript
+  // prior version 15.1.x
```
Comment (Lipata) [inline en/components/general/update-guide.md:62]:
```suggestion
  // version 16.1.x
```

File: en/components/general/update-guide.md [hunk 2]
```diff
@@ -51,6 +51,44 @@ Unfortunately not all changes can be automatically updated. Changes below are sp
...
+  public handleSelectionChanging(e: IComboSelectionChangingEventArgs): void {
+    this.selectedItems = e.newSelection;
+  }
+
+  // after version 15.1.x
```
Comment (Lipata) [inline en/components/general/update-guide.md:67]:
```suggestion
  // version 17.0.x
```

File: en/components/general/update-guide.md [hunk 3]
```diff
@@ -51,6 +51,44 @@ Unfortunately not all changes can be automatically updated. Changes below are sp
 
 For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
 
+## From 16.1.x to 17.0.x
+
```
Comment (Lipata) [inline en/components/general/update-guide.md:55]: Can you add a paragraph, describing the change from oldSelection/newSelection to oldValue/newValue and also the fact that there are migrations for that?

**Review by Lipata — CHANGES_REQUESTED** [review]

**Comment by hanastasov** [comment]
Closing in favour of https://github.com/IgniteUI/igniteui-docfx/pull/3980

**Review by Lipata — APPROVED** [review]

**Review by Lipata — APPROVED** [review]

**Pull request merged**